### ISSUE TITLE
Error creating runtime binarys with bdist-sage-linux

### ISSUE THREAD

**Issue by millermj1** [sagemath/binary-pkg#11]
If I clone https://github.com/sagemath/binary-pkg.git, edit sage.yaml to change the branch from develop to master, and run:

* $ export PACKAGE="Runtime binaries only"
* $ make bdist-sage-linux

then I get the errors:

* make stage-sage
* make[2]: Entering directory '/home/mike/Downloads/binary-pkg'
.../binary-pkg/activate python -m binary_pkg.cmdline --config sage.yaml --stage --package "Runtime binaries only"
* Namespace(build=False, checkout=False, config='sage.yaml', debug=False, dist=False, info=False, log=None, option_help=False, package='Runtime binaries only', stage=True)
* WARNING:root:Suspicious text terminators [b':'] in .../binary-pkg/source/SageMath/.../local/bin/gp-2.9
* WARNING:root:Contains zero-terminated strings but marker is not: .../binary-pkg/source/SageMath/.../local/bin/gp-2.9
* ERROR:root:At b'.../binary-pkg/source/SageMath/.../local/share/doc/sage/html/en/reference/calculus/sage/calculus/ode.htmlX\x1e'
* ERROR:root:path terminator b'\x1e' not allowed in .../binary-pkg/source/SageMath/.../local/share/doc/sage/doctrees/ca/intro/environment.pickle
* invalid string terminator
* Makefile:19: recipe for target 'stage-sage' failed
* make[2]: *** [stage-sage] Error 1

All the files in the directory .../binary-pkg/source/SageMath/.../local/share/doc/sage/html/en/reference/calculus/sage/calculus end in html .

Without the "export package" line, the compile succeeds, but I want to shrink the size as much as possible.

Thanks!

**Comment by vbraun** [comment]
Yes the runtime binary part isn't used regularly. The exclude: lines in sage.yaml need to be merged, anything at " Full binary tarball" needs to be added to "Runtime binaries only" if its not there already.
PR would be welcome..

**Comment by millermj1** [comment]
Thanks - I will try that.

In looking at sage.yaml, I notice that the rewrite_path begins with "&" in
the "Full binary tarball", but begins with "*" in the "Runtime binaries
only".  Should both begin with "&" ?

What PR would you like?

**Comment by vbraun** [comment]
Thats just YAML syntax, the rewrite_path is assigned (&) to the REWRITE_PATH_DEFAULT variable and referenced (*) later.

Only the files: section of "Runtime binaries only" needs updating....

**Comment by millermj1** [comment]
I'm new to YAML syntax, so please forgive my confusion.

If you're doing "Runtime binaries only", it seems like the assignment
"rewrite_path: &REWRITE_PATH_DEFAULT" in the section "Full binary tarball"
would not run, so that the later reference "rewrite_path:
*REWRITE_PATH_DEFAULT" in the section "Runtime binaries only" would be
undefined.

What I'm wondering is if I need to change to "rewrite_path:
&REWRITE_PATH_DEFAULT" in the section "Runtime binaries only".

Thanks

**Comment by vbraun** [comment]
No, the rewrite_path is correct as it stands. Only the files: section of "Runtime binaries only" needs updating....
